files/es/web/css/filter/index.md

```diff
@@ -30,1214 +28,192 @@ filter: opacity(25%);
+La propiedad `filter` se aplica a la segunda imagen, atenuando y difuminando tanto la imagen como su borde.
 
 ```css
-filter: sepia(100%);
-```
-
-```html hidden
-<table class="standard-table">
-  <thead>
-    <tr>
-      <th align="left" scope="col">Original image</th>
-      <th align="left" scope="col">Live example</th>
-      <th align="left" scope="col">SVG Equivalent</th>
-      <th align="left" scope="col">Static example</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td><img alt="Test_Form_12.jpeg" id="img1" class="internal default" src="/files/3727/Test_Form_12.jpeg" style="width: 100%;" /></td>
-      <td><img alt="Test_Form_12.jpg" id="img2" class="internal default" src="/files/3727/Test_Form_12.jpeg" style="width: 100%;" /></td>
-      <td><div class="svg-container"><svg xmlns="http://www.w3.org/2000/svg" id="img3" viewbox="0 0 259 194">
- <filter id="sepia">
-    <feColorMatrix type="matrix"
-               values="0.393 0.769 0.189 0 0
-                       0.349 0.686 0.168 0 0
-                       0.272 0.534 0.131 0 0
-                       0 0 0 1 0"/>
- </filter>
- <image xlink:href="/files/3727/Test_Form_12.jpeg" filter="url(#sepia)" width="259px" height="194px" />
-</svg><div></td>
-      <td><img alt="Test_Form_12_s.jpg" id="img4" class="internal default" src="/files/3728/Test_Form_12_s.jpg" style="width: 100%;" /></td>
-    </tr>
-  </tbody>
-</table>
-```
-
-```css hidden
-html {
-  height: 100%;
-}
-body {
-  font:
-    14px/1.286 "Lucida Grande",
-    "Lucida Sans Unicode",
-    "DejaVu Sans",
-    Lucida,
-    Arial,
-    Helvetica,
-    sans-serif;
-  color: rgb(51, 51, 51);
-  height: 100%;
-  overflow: hidden;
-}
-#img2 {
-  width: 100%;
-  height: auto;
-  -moz-filter: sepia(100%);
-  -webkit-filter: sepia(100%);
-  -o-filter: sepia(100%);
-  -ms-filter: sepia(100%);
-  filter: sepia(100%);
-}
-table.standard-table {
-  border: 1px solid rgb(187, 187, 187);
-  border-collapse: collapse;
-  border-spacing: 0px;
-  margin: 0px 0px 1.286em;
-  width: 85%;
-  height: 100%;
-}
-table.standard-table th {
-  border: 1px solid rgb(187, 187, 187);
-  padding: 0px 5px;
-  background: none repeat scroll 0% 0% rgb(238, 238, 238);
-  text-align: left;
-  font-weight: bold;
-}
-table.standard-table td {
-  padding: 5px;
-  border: 1px solid rgb(204, 204, 204);
-  text-align: left;
-  vertical-align: top;
-  width: 25%;
-  height: auto;
+img {
+  border: 5px solid yellow;
 }
-#img3 {
-  height: 100%;
+/* Ponga gris la segunda imagen en un 40% y desenfoque en 5px */
+img:nth-of-type(2) {
+  filter: grayscale(0.4) blur(5px);
 }
 ```
 
-{{EmbedLiveSample('sepia_example','100%','229px','')}}
+```html
+<img src="pencil.jpg" alt="La imagen original es nítida" />
+<img
+  src="pencil.jpg"
+  alt="La imagen y el borde están borrosos y silenciados." />
+```
+
+{{EmbedLiveSample('Applying_filter_functions','100%','229px')}}
```

Comment:
We should match [the heading](https://github.com/mdn/translated-content/pull/14992/files#diff-58885109584a3af1c4e2de2ebda18d2692939e592c43df78a366ba45ef2f718fR161), or use the id generated by Yari: `aplicar_funciones_de_filtro` (case insensitive).

You could get this through devtools:

![image](https://github.com/mdn/translated-content/assets/15844309/bb6c6d29-6bca-4872-9d46-8a3e78248c66)

Reference: [MDN Live sample](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Live_samples#embedlivesample_macro)

```suggestion
{{EmbedLiveSample('Aplicar funciones de filtro','100%','229px')}}
```

Reply:
Hi! Thanks for the review! @yin1999 

I remember from the conversation on removing macro transclusion that these parameter are not required anymore so you can use `{{EmbedLiveSample}}` without issues (as you can see [here](https://github.com/mdn/translated-content/blob/921baf1dc1827f133afd60f05f937d71cc57ef29/files/fr/web/api/batterymanager/level/index.md?plain=1#L38) from this [PR](https://github.com/mdn/translated-content/pull/6270)), and it seems that Yari throw flaws if the names are different, that's why I keep like that

Reply:
Yes, yari throws bugs, but I believe this is caused by requiring localized headings to use the same IDs as the English docs when we write the docs in HTML: https://github.com/mdn/translated-content/blob/0caa9937af24d77b83ea51de17d14673fd729f8e/docs/README.md?plain=1#L32-L40

Reply:
I don't know why it is, but this is the message when the parameter is different to `en-US`

![imagen](https://github.com/mdn/translated-content/assets/13079269/e5518327-5a44-4b42-ae59-83a32d84eea7)

Reply:
Hey @Graywolf9. Localization support in yari is slow at the moment, and I think we'll have to ignore with the flaws it throws for now. Because if the macro doesn't find the corresponding ID, it will contain the wrong code block (use the [closest heading instead](https://github.com/mdn/yari/blob/e09ed3bc39900f500a02e6a29d335926e0673d09/client/src/document/code/playground.ts#L161)), and I have encountered this problem many times (use the wrong id and broken the livesample), I've provided a example here: #15137.

The sender and receivers are all in the same iframe ([broken example](https://pr15137.content.dev.mdn.mozit.cloud/zh-CN/docs/Web/API/BroadcastChannel/message_event#%E7%BB%93%E6%9E%9C)):

![image](https://github.com/mdn/translated-content/assets/15844309/4649e6a9-3c5f-43e0-8bae-549a9bffdb61)

While they should be spread into three iframes ([correct example](https://developer.mozilla.org/zh-CN/docs/Web/API/BroadcastChannel/message_event#%E7%BB%93%E6%9E%9C)):

![image](https://github.com/mdn/translated-content/assets/15844309/7d7ba1e1-874f-443a-bd76-0cf13401f94c)

@queengooborg, may be we should remove the different id flaws report from [flaw system](https://github.com/mdn/yari/blob/main/build/flaws/translation-differences.ts), and should throw flaws if the incorrect id is used here (if set). This makes contributors use the wrong ids.

Reply:
Yeah, that flaw sounds like it's causing more issues than it's helping, not to mention we have a better way of tracking translation differences now.  Unfortunately, I don't have write access to Yari (and my PRs are considered "community" PRs...even though I'm a part of the MDN team...), so I cannot make those changes.  :(

Reply:
Thank you so much for the example! @yin1999 As I can see there is a problem only when the macro is used more than once in a header

Thank you so much again!